### DISCUSSION TITLE
Tasmota devices and Mi Home

### DISCUSSION THREAD

**Discussion by lyolic** [arendst/Tasmota#10396] [Q&A] [answered]
Hi All, Are there any ability to use devices with Tasmota firmware with Mi Home?
As far as I can see Tasmota has Belkin WeMo and Hue Bridge emulation. Can we use it to connect tasmota devices to cloud?

**Comment by Jason2866** [comment, marked as answer]
First of all Tasmota is designed for use with mqtt. WemMo and HueBridge emulation is added because it makes it easy possible
to use it with Amazon Alexa device(s) in the LAN.  There is no security risk since NO Tasmota device is connected directly to the internet nor is it reachable. If Mi Home can uses services from Alexa API you can try to connect with.
A other feasable way could be connecting Tasmota to AWS IoT service and build a skill here to connect to MiHome.
The Tasmota setup for AWS IoT is in the documentation.
